pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExporter.java

```diff
@@ -90,7 +90,12 @@
         List<CustomExportParticipant> customExportParticipants = new ArrayList<>();
         for (Participant p : participants) {
             String familyId = getFamilyId(handle, studyDto, p.getUser().getGuid(), esClient);
-            customExportParticipants.add(new CustomExportParticipant(familyId, p));
+            if (familyId != null) {
+                customExportParticipants.add(new CustomExportParticipant(familyId, p));
+            } else {
+                LOG.error("Exclude participant with GUID = '{}' from the export batch: "
+                        + "Failed to get family ID for the participant because dsm.familyId is null", p.getUser().getGuid());
```

Comment:
LOG.error("Skipped participant with GUID = '{}' from the export batch because dsm.familyId is null ", p.getUser().getGuid());